### PR TITLE
Todo feature

### PR DESCRIPTION
**Pull request by Mohitkumar33**
To do feature complete of extension. 
Add todos
remove todos
Save todos 
Show todos on refresh

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { LandingPage } from "./components/landingPage";
```
Comment (VedanthB) [inline src/App.js:1]: you can barrel import/export

**Comment by Mohitkumar33** [comment]
deploy preview
https://deploy-preview-5--focus-extension.netlify.app/

**Review by VedanthB — APPROVED** [review]
Good work 

- clean code and organised folder structure
- good variable names

You can merge the PR

File: src/App.js (hunk 1 above)
Comment (Mohitkumar33) [reply src/App.js:1]: ok Thanks

File: src/components/small components/todoAdded.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        onChange={(e) => setTodoInput(e.target.value)}
+        onKeyPress={(e) =>
+          e.key === "Enter"
+            ? (setTodosList([
+                ...todosList,
+                { id: uuid(), todoNote: todoInput, isDone: false },
+              ]),
+              setTodoInput(""),
+              localStorage.setItem("todoTasks", JSON.stringify(todosList)))
+            : undefined
```
Comment (prathmeshjagtap) [inline src/components/small components/todoAdded.js:90]: you can write null instead of undefined

**Review by prathmeshjagtap — APPROVED** [review]
Code is Well structured 
Good Naming Conventions
Clean Code

**Pull request merged**